lemarche/www/tenders/views.py

```diff
@@ -98,6 +104,16 @@ class TenderDetail(LoginRequiredMixin, DetailView):
     template_name = "tenders/detail.html"
     context_object_name = "tender"
 
+    def get(self, request, *args, **kwargs):
+        """
+        Check if the User has any Siae contacted for this Tender. If yes, update 'detail_display_date'
+        """
+        tender = self.get_object()
+        TenderSiae.objects.filter(
+            tender=tender, siae__in=self.request.user.siaes.all(), detail_display_date=None
+        ).update(detail_display_date=timezone.now())
```

Comment:
En faisant le check sur le type de user, ça nous évitera une requête systématique pour tout les utilisateurs :) 
```suggestion
        if self.request.user.kind ==  User.KIND_SIAE:
            tender.siaes.filter(
                siae__in=self.request.user.siaes.all(), detail_display_date=None
            ).update(detail_display_date=timezone.now())
```

Reply:
T'es sûr que ca marche ? vu qu'on souhaite maj un champ sur `tender.tendersiae_set` et non `tender.siaes`. je vais regarder

Reply:
ok j'ai rajouté le check :)